src/app/pages/Login.js

```diff
@@ -7,7 +7,10 @@ import { AuthContext } from '../../backend/Auth';
 
 function Login() {
     const navigate = useNavigate()
-    const signInWithGoogle = (event) => {
+    const { currentUser } = useContext(AuthContext)
+    const loading = typeof currentUser === 'undefined'
+
+    function signInWithGoogle(event) {
```

Comment:
Adding a load state indicator & switching the signInWithGoogle variable to use the function keyword for clarity